warehouse/testhelper/setup.go

```diff
@@ -458,20 +474,49 @@ func BRTJobsForUserId() string {
 `
 }
 
-func DefaultEventMap() EventsCountMap {
-	return EventsCountMap{
-		"identifies":    1,
-		"users":         1,
-		"tracks":        1,
-		"pages":         1,
-		"product_track": 1,
-		"screens":       1,
-		"aliases":       1,
-		"groups":        1,
-		"_groups":       1,
-		"gateway":       6,
-		"batchRT":       8,
+func DefaultEventMap(withSources bool) EventsCountMap {
```

Comment:
Let's not use conditions here. Instead create a separate default for SourcesEventsMap.